src/components/Layout/NavBar.component.jsx

```diff
@@ -0,0 +1,111 @@
+import {
+  AppBar,
+  IconButton,
+  fade,
+  makeStyles,
+  Toolbar,
+  Typography,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import SearchIcon from '@material-ui/icons/Search';
+import InputBase from '@material-ui/core/InputBase';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import Switch from '@material-ui/core/Switch';
+import React from 'react';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
I'm not familiar on how material-ui `makeStyles` work, but perhaps moving this outside the component would be better. For what I can see, it lets you put some theming to the component.